### DISCUSSION TITLE
Taker news feed

### DISCUSSION THREAD

**Discussion by rishflab** [get10101/itchysats#1133] [Ideas]
I think we should have a "live tile" or feed which displays news/messages/urgent updates to takers in case something goes wrong on our end or with our software.

An example where this could of been useful was when we encountered the "not enough utxo's" issue on jan18th. We could of informed users that we are experiencing more users than we expected and have run out of utxos. I think this would be reassuring for users. It also might reduce the number of issues and nonsense complaints we get on telegram etc.

**Comment by bonomat** [comment]
Archiving this discussion due to inactivity.
